extensions/wikia/AdEngine/js/tracking/adInfoListener.js

```diff
@@ -90,10 +91,12 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 			slotPrices.a9 = vastInfo.amznbid;
 		}
 
+		slotRegistry.storeScrollY(vastInfo.pos);
```

Comment:
We use this value for analysis around better targeting of ads. It means that the value should be calculated before we perform DFP request. It means that we should change how and when we call this method for video.

I'd suggest to call this method from https://github.com/Wikia/app/blob/fc4e1e880fcd072f66959723f32e76b114455d2f/extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js